tests/config_test.py

```diff
@@ -15,31 +15,31 @@ def test_pollfile_argument_overrides_pollfile_defined_in_file(self, zino_conf, p
         assert config
         assert config.polling.file == str(polldevs_conf_with_single_router)
 
-    def test_raises_error_on_file_not_found(self):
-        with pytest.raises(OSError):
-            read_configuration("non-existent-config.toml")
+    def test_raises_error_on_file_not_found(self, tmp_path):
+        with pytest.raises(OSError) as excinfo:
+            read_configuration(tmp_path / "non-existent-config.toml")
+
+        assert excinfo.type is FileNotFoundError
```

Comment:
If the intention is to only match `FileNotFoundError`s, why not just:

```suggestion
        with pytest.raises(FileNotFoundError) as excinfo:
            read_configuration(tmp_path / "non-existent-config.toml")
```

?

Reply:
Good question! I tried to do fancy stuff I learned at EuroPython, but somehow did not realize that there are much easier ways :sweat_smile: